specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json

```diff
@@ -269,66 +269,6 @@
             }
           }
         }
-      },
-      "patch": {
```

Comment:
>patch [](start = 7, length = 5)

Clusters are a tracked resource, which requires supporting a PATCH operation that allows the user to at least update tags. Why are you not supporting this?

Reply:
We reconsidered and will support PATCH. Closing the PR.